plugin.video.odeon/odeon.py

```diff
@@ -452,28 +460,33 @@ def play(params):
     response = requests.get(url, headers=get_headers(auth=utils.digest(clave)))
     data, errmsg = decode_json(response)
     if data:
-        if params.has_key('context'):
-            # acceso por menu contextual debe setear metadata
-            path = '{0}/prod/{1}?perfil={2}'.format(params['source'], params['sid'], PID)
-            prod = json_request(path)
-
-            info = get_info(prod)
-            item = xbmcgui.ListItem(label=info['title'], path=data["url"])
-            item.setInfo('video', info)
-            item.setArt(get_art(prod))
-
-            item.addStreamInfo('video', {'codec': 'h264'})
-            item.addStreamInfo('audio', {'codec': 'aac', 'language' : 'es'})
-
-            xbmc.Player().play(data["url"], item)
-            seek = data.get('seek')
+        response = requests.get(data["url"], headers=get_headers(auth=utils.digest(clave)))
+        if response.status_code == 200:
+            if params.has_key('context'):
+                # acceso por menu contextual debe setear metadata
+                path = '{0}/prod/{1}?perfil={2}'.format(params['source'], params['sid'], PID)
+                prod = json_request(path)
+
+                info = get_info(prod)
+                item = xbmcgui.ListItem(label=info['title'], path=data["url"])
+                item.setInfo('video', info)
+                item.setArt(get_art(prod))
+
+                item.addStreamInfo('video', {'codec': 'h264'})
+                item.addStreamInfo('audio', {'codec': 'aac', 'language' : 'es'})
+
+                xbmc.Player().play(data["url"], item)
```

Comment:
Can you please use xbmcplugin.setresolvedurl here instead of creating a player instance? Plugins should not use the player directly, only scripts

Reply:
Done. Upgraded to version 1.3.4